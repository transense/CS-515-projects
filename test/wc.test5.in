"C:\Users\vaibh\OneDrive\Documents\CS 515
 A(fundamentals of Computing)\
        Project_1\CS-515-projects\wc_test4.txt"