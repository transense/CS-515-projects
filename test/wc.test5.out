     3   6   120 test/wc.test5.in